tests/integration/test_provision_int.py

```diff
@@ -83,7 +108,7 @@ def test_provision_interrupt_child(initproj, monkeypatch, capfd, signal_type):
             "tox.ini": """
                     [tox]
                     skipsdist=True
-                    minversion = 3.7.0
+                    minversion = 3.7.0,<4
```

Comment:
I don't know if this is explicitly supported but thankfully it works. Without it the tests were failing (as they were provisioning tox 4).